appveyor.yml

```diff
@@ -1,7 +1,7 @@
-
+image: Visual Studio 2019
 environment:
   matrix:
-    - PYTHON: "C:\\Python36-x64"
+    - PYTHON: "C:\\Python38-x64"
```

Comment:
is it interesting to test other python versions?
otherwise, we can remove `matrix`.
If we test multiple python versions we should also add the version to the name of the file generated as artifact

Reply:
Good call! Travis currently tests 3.6, 3.7 and 3.8 so I think only 3.8 is enough in AppVeyor.

Removed the `matrix:` line.